src/main/java/it/eng/idsa/businesslogic/service/impl/TransportCertsManager.java

```diff
@@ -67,7 +67,6 @@ private void populateTransportCertsSha() {
 			}
 		});
 		transportCerts.remove("localhost");
-		logger.info("Calculated {} transportCertsSha256 values", transportCerts.entrySet().size());
 		if (logger.isDebugEnabled()) {
```

Comment:
This might consider as sensitive data, to print transportCertsSha256 , even in debug mode

Reply:
Removed